src/index.html

```diff
@@ -5,9 +5,57 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,700&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div class="imageImac"></div>
+
+      <div class="nameImac">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+
+      <div class="codeImac">
+        Product code: 195434
+      </div>
+
+      <div class="starsAndReviews">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="reviews">
+          Reviews: 5
+        </div>
+      </div>
+
+      <div class="priceAndCost">
+        <div class="price">
+          Price:
+        </div>
+        <div class="cost">
+          $2,199
+        </div>
+      </div>
+
+      <a
+        href="#"
+        class="buttonBuy"
+        data-qa="hover"
+      >Buy</a>
```

Comment:
```suggestion
      >
        Buy
      </a>
```
Fix indents according to the code style guide